apps/blueprints/src/pages/api/blueprint/delete/[id].ts

```diff
@@ -0,0 +1,29 @@
+import { deleteBlueprintPage } from "@factorio-sites/database";
+import { parseDatabaseError } from "../../../../utils/api.utils";
+import { apiHandler } from "../../../../utils/api-handler";
+
+const handler = apiHandler(async (req, res, { session }) => {
+  if (req.method !== "DELETE") return res.status(400).json({ error: "method must be DELETE" });
+
+  if (!session) return res.status(401).json({ status: "Not authenticated" });
+
+  const { id } = req.query;
+  if (!id) return res.status(400).json({ status: "ID is required" });
+
+  try {
+    await deleteBlueprintPage(String(id));
```

Comment:
Lets query the blueprint first to make sure it's user_id matches session.user_id